web/src/pages/CreateRoom/CreateRoom.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, createSignal, Show } from "solid-js";
+import Button from "@/components/Button";
+
+import styles from "./CreateRoom.module.scss";
+import useCreateRoom from "./hooks/useCreateRoom";
+
+const defaultNumberOfOptions = 5;
+
+const CreateRoom: Component = () => {
+	const { fields, updateField, submit } = useCreateRoom();
+
+	const handleSubmit = (event: Event): void => {
+		event.preventDefault();
+		submit();
+	};
+
+	return (
+		<main class={styles.createRoom}>
+			<form class={styles.createRoomForm} onSubmit={handleSubmit}>
+				<label for="moderatorName">
+					Your name
+					<span class={styles.required}>*</span>
+				</label>
+
+				<input name="moderatorName" type="text" onInput={updateField} />
+
+				<div class={styles.seperator}>
+					<hr />
+				</div>
+
+				<label for="voterOptions">
+					What options would you like voters have to choose from?
+					<span class={styles.required}>*</span>
+				</label>
+
+				<select name="voterOptions" onInput={updateField}>
+					<option value="select">Select options</option>
+					<option value="fibonacci">Fibonacci</option>
+					<option value="linear">Linear</option>
+				</select>
+
+				<label for="numberOfOptions">Number of options</label>
+
+				<input
+					type="range"
+					name="numberOfOptions"
+					min="2"
+					max="15"
+					value={fields.numberOfOptions}
+					step="1"
+					onInput={updateField}
+				/>
+
```

Comment:
I left off the Include no vote checkboxes since there wasn't anywhere to put it in the db. So I'll make a new issue to add that in next

Reply:
I believe that the "No Vote" option should just include `0` in the array of options. That way we don't have to store another property, just the vote options.

Reply:
Ah that makes sense! I didn't even think about 0 as being that value. Cool I'll update that in my PR 👍

Reply:
- [ ] add no vote checkbox